openapi/components/paths/instances.yaml

```diff
@@ -100,25 +100,6 @@ paths:
           $ref: ../responses/instances/InstanceShortNameResponse.yaml
         '401':
           $ref: ../responses/MissingCredentialsError.yaml
-  '/instances/{worldId}:{instanceId}/invite':
-    parameters:
-      - $ref: ../parameters.yaml#/worldId
-      - $ref: ../parameters.yaml#/instanceId
-    post:
-      summary: Send Self Invite
-      tags:
-        - instances
-      operationId: sendSelfInvite
-      security:
-        - authCookie: []
-      description: Sends an invite to the instance to yourself.
-      x-codeSamples:
-        $ref: "../codeSamples/instances.yaml#/~1instance~1{worldId}:{instanceId}~1invite/post"
```

Comment:
Can you also delete the related files?

Reply:
I deleted the example and the path

Reply:
This one as well.
``$ref: ../responses/instances/InstanceSelfInviteSuccess.yaml``

Reply:
https://github.com/vrchatapi/specification/pull/391/commits/b8c05878ff59de865c20ca77dd69f60a1aade3e3
Didnt i delete that one here?

Reply:
Ah, was reviewing the PR on the GitHub mobile app, must've missed that. Did you run your changes against the test repository?

Reply:
No, good shout, will do

Reply:
Done, should have fixed the issues that exposed